packages/client/src/views/RegisterForm/RegisterForm.tsx

```diff
@@ -714,6 +706,13 @@ class RegisterFormView extends React.Component<FullProps, State> {
         }
       }
       this.props.modifyDeclaration(modifiedDeclaration)
+      // this is to forcefully remount the component
+      // to reset the initial values of formik
+      this.setState({
+        formFieldKey: `${this.props.activeSection.id}-${
+          this.props.activeSectionGroup.id
+        }-${Date.now()}`
```

Comment:
can there be another way of force-rerender? May be a different key for different informant type would fit here